modules/nf-core/samtools/bam2fq/main.nf

```diff
@@ -21,14 +21,18 @@ process SAMTOOLS_BAM2FQ {
     script:
     def args = task.ext.args ?: ''
     def prefix = task.ext.prefix ?: "${meta.id}"
-    def single_end = reads.collect().size() == 1
 
-    if (single_end) {
+    if (split) {
```

Comment:
so this could either be `if (!single_end)` or you do `if(single_end)` and switch the order of the conditionals